### PR TITLE
fix(ci): use normal docker to build docker images, instead of buildx.

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind bug

**Any specific area of the project related to this PR?**

/area CI

**What this PR does / why we need it**:

We are being hit by https://github.com/docker/buildx/issues/1758.
Avoid using buildx (we aren't using any feature btw)

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

Refs  #2501 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by FedeDP** [comment]
cc @LucaGuerra

File: .github/workflows/reusable_build_docker.yaml [hunk 1]
```diff
@@ -36,41 +36,35 @@ jobs:
...
-          build-args: |
-            VERSION_BUCKET=deb${{ inputs.bucket_suffix }}
-            FALCO_VERSION=${{ inputs.version }}
-          tags: |
-            docker.io/falcosecurity/falco:${{ inputs.arch }}-${{ inputs.tag }}
-          outputs: type=docker,dest=/tmp/falco-${{ inputs.arch }}.tar
+        run: |
+          cd ${{ github.workspace }}/docker/falco/
+          docker build -t docker.io/falcosecurity/falco:${{ inputs.arch }}-${{ inputs.tag }} \
+            --build-arg VERSION_BUCKET=deb${{ inputs.bucket_suffix }}
```
Comment (LucaGuerra) [inline .github/workflows/reusable_build_docker.yaml:51]: missing `\` at the end of the line?

File: .github/workflows/reusable_build_docker.yaml [hunk 2]
```diff
@@ -36,41 +36,35 @@ jobs:
...
-          context: ${{ github.workspace }}/docker/driver-loader/
-          build-args: |
-            FALCO_IMAGE_TAG=${{ inputs.arch }}-${{ inputs.tag }}
-          tags: |
-            docker.io/falcosecurity/falco-driver-loader:${{ inputs.arch }}-${{ inputs.tag }}
-          outputs: type=docker,dest=/tmp/falco-driver-loader-${{ inputs.arch }}.tar  
+        run: |
+          cd ${{ github.workspace }}/docker/driver-loader/
+          docker build -t docker.io/falcosecurity/falco-driver-loader:${{ inputs.arch }}-${{ inputs.tag }} \
+            --build-arg FALCO_IMAGE_TAG=${{ inputs.arch }}-${{ inputs.tag }}
```
Comment (LucaGuerra) [inline .github/workflows/reusable_build_docker.yaml:65]: missing `\` at the end of the line?

Comment (FedeDP) [reply .github/workflows/reusable_build_docker.yaml:65]: Thanks!

Comment (FedeDP) [reply .github/workflows/reusable_build_docker.yaml]: Fixed

**Review by LucaGuerra — APPROVED** [review]
LGTM

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**